test/verify/check-openshift

```diff
@@ -652,7 +652,7 @@ class TestRegistry(MachineCase):
             b.click(".btn-primary")
             b.wait_not_present("modal-dialog")
             b.wait_present("table.listing-ct")
-            wait(lambda: username not in o.execute("oc get rolebinding -n testprojectuserproj"))
+            wait(lambda: username not in o.execute("oc get rolebinding -n testprojectuserproj"), delay=5)
```

Comment:
Agreed, increasing the polling delay isn't much harm (a few extra seconds on a local run, but *shrug*). If it helps, that's great, I was just wondering if that's really the cause. Being over a thousand times slower than on a local run smells fishy.